src/NuGet/BuildNuGets.csx

```diff
@@ -87,6 +87,7 @@ string[] RedistPackageNames = {
     "Microsoft.CodeAnalysis.EditorFeatures",
     "Microsoft.CodeAnalysis.EditorFeatures.Text",
     "Microsoft.CodeAnalysis.Features",
+    "Microsoft.CodeAnalysis.Remote.Razor.ServiceHub",
```

Comment:
LCA is generally your best bet for these questions.

@Pilchie might know if we have any other contacts or existing rules for these shim libraries (which I expect we don't want other using to get around IVTs).